db/migrate/20230510150855_change_status_to_is_canceled.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeStatusToIsCanceled < ActiveRecord::Migration[6.1]
+  def change
+    add_column(:appointments, :is_canceled, :boolean, default: false, null: false)
+    Appointment.status_canceled.update_all("is_canceled=TRUE")
+    Appointment.status_pending.update_all("is_canceled=FALSE")
```

Comment:
Unfortnuetally I had trouble applying the migration because it was written when field `status` in Appointment model was available. But since you have removed it, we cannot use it anymore 😞 so it must be filtered manually. It's essential to not repeat this in the future because I haven't thought about it before.

```suggestion
    Appointment.where("status=0").update_all("is_canceled=FALSE")
    Appointment.where("status=1").update_all("is_canceled=TRUE")
```